### PR TITLE
[Docs] fixes broken link 

### PR DESCRIPTION
**Pull request by MarvinGravert**
Hi,

i have noticed that the link to the schema which is referenced in [`contributing.md`](https://github.com/devcontainers/spec/blob/main/contributing.md?plain=1#L17) and [`devcontainerjson-reference.md`](https://github.com/devcontainers/spec/blob/main/docs/specs/devcontainerjson-reference.md?plain=1#L164) is broken. It links to a `devContainer.schema.src.json` file that no longer exists in the vscode repo (removed in this [PR](https://github.com/microsoft/vscode/pull/169857)).

I have adjusted the docs to link to the [`devContainer.base.schema.json`](https://github.com/devcontainers/spec/blob/main/schemas/devContainer.base.schema.json) which is found in this repo.

I hope this is correct. If not, sorry for any inconvenience.

### CODE REVIEW

**Comment by MarvinGravert** [comment]
@microsoft-github-policy-service agree

**Comment by MarvinGravert** [comment]
Thanks for your feedback @bamurtaugh ! I have changed the link to your recommandation.

File: contributing.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ Here is a sample:
 |----------|------|-------------|
 | `image` | string | **Required** when [using an image](/docs/remote/create-dev-container.md#using-an-image-or-dockerfile). The name of an image in a container registry ([DockerHub](https://hub.docker.com), [GitHub Container Registry](https://docs.github.com/packages/guides/about-github-container-registry), [Azure Container Registry](https://azure.microsoft.com/services/container-registry/)) that VS Code and other `devcontainer.json` supporting services / tools should use to create the dev container. |
 
-- PRs to the [schema](https://github.com/microsoft/vscode/blob/main/extensions/configuration-editing/schemas/devContainer.schema.src.json), i.e code or shell scripts demonstrating approaches for implementation.
+- PRs to the [schema](https://github.com/devcontainers/spec/blob/main/schemas/devContainer.base.schema.json), i.e code or shell scripts demonstrating approaches for implementation.
```
Comment (bamurtaugh) [inline contributing.md]: My mistake on this one - since this is talking about the spec overall rather than the VS Code specific spec (as the other link points to), I think your first update to `https://github.com/devcontainers/spec/blob/main/schemas/devContainer.base.schema.json` would make sense here!

Comment (MarvinGravert) [reply contributing.md:17]: Ah great! I have reverted my last commit

File: docs/specs/devcontainerjson-reference.md [hunk 2]
```diff
@@ -161,7 +161,7 @@ Variables can be referenced in certain string values in `devcontainer.json` in t
 
 ## Schema
 
-You can see the VS Code implementation of the dev container schema [here](https://github.com/microsoft/vscode/blob/main/extensions/configuration-editing/schemas/devContainer.schema.src.json).
+You can see the VS Code implementation of the dev container schema [here](https://github.com/devcontainers/spec/blob/main/schemas/devContainer.base.schema.json).
```
Comment (bamurtaugh) [inline docs/specs/devcontainerjson-reference.md:164]: Since this is no longer pointing to the VS Code specific schema, would suggest rewording:

```suggestion
You can see the dev container schema [here](https://github.com/devcontainers/spec/blob/main/schemas/devContainer.base.schema.json).
```

**Review by Adityarujid30 — CHANGES_REQUESTED** [review]
https://my.1password.com/sign-up/verify/eyJhY2NvdW50VHlwZSI6ImZhbWlseSIsInR5cGUiOiJGIiwidXNlck5hbWUiOiJHaGFsaWIgUnVqaXRvIiwiZW1haWwiOiJhZGl0eWEuYXByaWxAcm9ja2V0bWFpbC5jb20iLCJlbWFpbFJlc2VuZFRva2VuIjoiNk1IMks3U0hTQkJZNUhTTFdRVFpHSFRFNk0iLCJuZXdzbGV0dGVyU3Vic2NyaWJlIjp0cnVlLCJjb21wYW55U2l6ZSI6IiIsInB1cmNoYXNlT3JkZXIiOiIiLCJzdHJpcGVQdXJjaGFzZU9yZGVyIjoiIiwiYWNjb3VudE5hbWUiOiJPdXIgRmFtaWx5In0?l=enhttps://www.facebook.com/aditya.april?mibextid=ZbWKwL

**Review by samruddhikhandale — APPROVED** [review]
Thank you!

**Review by Adityarujid30 — APPROVED** [review]
